docs/sources/datasources/mysql/query-editor/_index.md

```diff
@@ -184,106 +184,127 @@ The examples in this section refer to the data in the following table:
+| 2025-01-02 03:10:00 | 6.0          | 2025-01-02 03:10:00 | 10.0.1.1 |
+| 2025-01-02 03:11:00 | 7.0          | 2025-01-02 03:11:00 | 10.0.1.2 |
+| 2025-01-02 03:20:00 | 5.0          | 2025-01-02 03:20:00 | 10.0.1.2 |
 +---------------------+--------------+---------------------+----------+
 ```
 
-A time series query result is returned in a [wide data frame format](https://grafana.com/developers/plugin-tools/key-concepts/data-frames#wide-format). Any column except time or of type string transforms into value fields in the data frame query result. Any string column transforms into field labels in the data frame query result.
-
 {{< admonition type="note" >}}
 For backward compatibility, an exception to the aforementioned rule applies to queries returning three columns, including a string column named `metric`. Instead of converting the metric column into field labels, it is used as the field name, and the series name is set to the value of the metric column. Refer to the following example with a metric column.
 {{< /admonition >}}
 
-**Example with `metric` column:**
+**Example with `$__time(dateColumn)` Macro:**
+
+```sql
+SELECT
+  $__time(time_date_time),
+  value_double
+FROM my_data
+ORDER BY time_date_time
+```
+
+Table panel result:
+
+{{< figure alt="output of time macro" src="/media/docs/grafana/data-sources/mysql/screenshot-time-and-timefilter-macro.png" >}}
+
+In the following example, the result includes two columns, `Time` and `value_double`, which represent the data associated with fixed timestamps. This query does not apply a time range filter and returns all rows from the table.
+
+**Example with `$__timeFilter(dateColumn)` Macro:**
+
+```sql
+SELECT
+  $__time(time_date_time),
+  value_double
+FROM my_data
+WHERE $__timeFilter(time_date_time)
+ORDER BY time_date_time
+```
+
+Table panel result:
+
+{{< figure alt="output of time filter macro" src="/media/docs/grafana/data-sources/mysql/screenshot-time-and-timefilter-macro.png" >}}
+
+This example returns the same result as the previous one, but adds support for filtering data using the Grafana time picker.
+
+**Example with `$__timeGroup(dateColumn,'5m')` Macro:**
+
+```sql
+SELECT
+  $__timeGroup(time_date_time, '5m') AS time,
+  sum(value_double) AS sum_value
+FROM my_data
+WHERE $__timeFilter(time_date_time)
+GROUP BY time
+ORDER BY time
+```
+
+Table panel result:
+
+{{< figure alt="output of time group macro" src="/media/docs/grafana/data-sources/mysql/screenshot-timegroup-macro.png" >}}
+
+Given the result in the following example, the data is grouped and aggregated within buckets with timestamps of fixed interval i.e. 5 mins. To customize the default series name formatting (optional), refer to [Standard options definitions](https://www.google.com/url?q=https://grafana.com/docs/grafana/latest/panels-visualizations/configure-standard-options/&sa=D&source=docs&ust=1752744133031256&usg=AOvVaw3WN6JEG7QlBgFC6HPNnS_U).
```

Comment:
What is this url? 😳

Reply:
@usmangt all the URLs need to be replaced with internal links and instead of latest you'll need to use [version substitution](https://grafana.com/docs/writers-toolkit/write/shortcodes/#about-version-substitution) (essentially replace `latest` with `<GRAFANA_VERSION>`).